lib/v4/migration-helpers/update-api-folder-structure/templates/core-service.liquid

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+/**
+ * {{ id }} service
```

Comment:
I think these commands would look nicer if we used the display name instead of id

Reply:
Oh I was basing it on what we currently do with the existing generators I think :thinking: Though I do agree with you, it might look weird when there are differences between migrated files vs new ones.